sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsExtensionConfigProvider.cs

```diff
@@ -205,6 +207,39 @@ public async Task<HttpResponseMessage> ConvertAsync(HttpRequestMessage input, Ca
             return (T)blob.BlobClient;
         }
 
+        private ParameterBindingData ConvertToParameterBindingData(BlobAttribute blobAttribute)
+        {
+            var blobPath = BlobPath.ParseAndValidate(blobAttribute.BlobPath);
+
+            var blobDetails = new BlobParameterBindingDataContent()
+            {
+                Connection = _nameResolver.Resolve(blobAttribute.Connection) ?? Constants.DefaultAzureStorageConnectionName,
+                BlobName = blobPath.BlobName,
+                ContainerName = blobPath.ContainerName
+            };
+
+            return CreateParameterBindingData(blobDetails);
+        }
+
+        private ParameterBindingData ConvertToParameterBindingData(BlobBaseClient input, BlobTriggerAttribute blobTriggerAttribute)
+        {
+            var blobDetails = new BlobParameterBindingDataContent()
```

Comment:
Seems like we can remove some code duplication by having the CreateParameterBindingData method take connection/blobname/containername parameters, and have it new up the details itself. The connection name resolver stuff, is centralized there as well.

Reply:
Done